packages/ui/react-ui-table/src/model/table-model.ts

```diff
@@ -118,6 +120,7 @@ export class TableModel<T extends TableRow = TableRow> extends Resource {
   constructor({
     view,
     schema,
+    space,
```

Comment:
why is this necessary? why can you not just use the space of the view? this makes the model more difficult to use as evidenced by needing to cast the props below. It seems perfectly valid to have a table model outside a space (e.g. dynamic tables)